docs/src/linter/github-actions.md

```diff
@@ -1,21 +1,21 @@
 # Github Actions
 
-`bevy_lint` provides an action to conveniently install the linter in CI:
+> **Important**
+>
+> The action is only available for versions v0.3.0 and onward. v0.2.0 and v0.1.0 will not work, however you may emulate it by manually running the [installation commands](install.md) in your workflow.
```

Comment:
In the long term (and in a separate PR) we should define some sort of shortcut or component to indicate version compatibility in the book, something like "since v0.3.0".

This would also be useful for the CLI!